### DISCUSSION TITLE
在 Select 选项改变时，获取改变前后的值

### DISCUSSION THREAD

**Discussion by DejanHuang** [dotnetcore/BootstrapBlazor#3817] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

需要在改变选项时，同步更新一些其他的数据，但是 OnSelectedItemChanged 无法获取到修改前的值，使用 OnBeforeSelectedItemChange 又会有弹窗，如果在这里更新，弹窗中选择取消就不对了。

### Describe the solution you'd like

OnSelectedItemChanged 中能获取到改变前的值，或者 OnBeforeSelectedItemChange 能设置不弹窗。

**Comment by bb-auto** [comment]
@huangyy21 Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply]
完全不知道你要表达什么
